03week/towersOfHanoi.js

```diff
@@ -2,49 +2,90 @@
 
 const assert = require('assert');
 const readline = require('readline');
-const rl = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout
-});
+const rl = readline.createInterface({input: process.stdin, output: process.stdout});
+let removedNumber = 0;
+let pushedNumber = 0;
 
 let stacks = {
-  a: [4, 3, 2, 1],
+  a: [
+    4, 3, 2, 1
+  ],
   b: [],
   c: []
 };
 
+function resetGame(reset) {
+  if (reset === 'yes') {
+    stacks = {
+      a: [
+        4, 3, 2, 1
+      ],
+      b: [],
+      c: []
+    };
+    getPrompt()
+  } else {
+    console.log('ctrl + c to quit')
+
+  }
+}
 function printStacks() {
   console.log("a: " + stacks.a);
   console.log("b: " + stacks.b);
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
+function movePiece(startStack, endStack) {
```

Comment:
the only thing that this function should do is move a piece. It shouldn't check for legality.